### ISSUE TITLE
Adding Beak as a package dependency

### ISSUE THREAD

**Issue by jakeheis** [yonaskolb/Beak#18]
I'm trying to add Beak as a dependency to my project and import the `BeakCore` library, but I get this error when building:

```
Fetching https://github.com/yonaskolb/Beak
error: the package https://github.com/yonaskolb/Beak @ 0.3.3 contains revisioned dependencies:
    https://github.com/yonaskolb/swift-package-manager @ untyped_arguments
error: product dependency 'BeakCore' not found
```

I believe to fix this issue, https://github.com/yonaskolb/swift-package-manager needs to have a new version tagged on the `untyped_arguments` branch, and then the `Package.swift` in this repo needs to be changed from:

```swift
...
.package(url: "https://github.com/yonaskolb/swift-package-manager", .branch("untyped_arguments")),
...
```

to:

```swift
...
.package(url: "https://github.com/yonaskolb/swift-package-manager", from: "new-version"),
...
```

**Comment by yonaskolb** [comment]
Thanks for pointing this out @jakeheis.
I've fixed this by linking to the upstream apple repo which has my required changes now merged in #19 
It's in master and will be in the next release 👍

**Comment by yonaskolb** [comment]
Released in 0.3.4

**Comment by jakeheis** [comment]
Thanks for the quick fix!
